diff -rupN ori/servers/slapd/overlays/Makefile.in openldap-2.4.44/servers/slapd/overlays/Makefile.in
--- ori/servers/slapd/overlays/Makefile.in	2017-07-31 22:02:20.695985336 +0200
+++ openldap-2.4.44/servers/slapd/overlays/Makefile.in	2017-07-31 22:06:14.789579678 +0200
@@ -33,6 +33,7 @@ SRCS = overlays.c \
 	syncprov.c \
 	trace.c \
 	translucent.c \
+	twcompare.c \
 	unique.c \
 	valsort.c \
 	smbk5pwd.c \
@@ -56,7 +57,7 @@ NT_LINK_LIBS = -L.. -lslapd $(@BUILD_LIB
 UNIX_LINK_LIBS = $(@BUILD_LIBS_DYNAMIC@_LDAP_LIBS)
 
 LIBRARY = ../liboverlays.a
-PROGRAMS = @SLAPD_DYNAMIC_OVERLAYS@ smbk5pwd.la allop.la trace.la
+PROGRAMS = @SLAPD_DYNAMIC_OVERLAYS@ smbk5pwd.la allop.la trace.la twcompare.la
 
 XINCPATH = -I.. -I$(srcdir)/..
 XDEFS = $(MODULES_CPPFLAGS)
@@ -143,6 +144,8 @@ allop.la : allop.lo
 trace.la : trace.lo
 	$(LTLINK_MOD) -module -o $@ trace.lo version.lo $(LINK_LIBS)
 
+twcompare.la: twcompare.lo
+	$(LTLINK_MOD) -module -o $@ twcompare.lo version.lo $(LINK_LIBS)
 
 install-local:	$(PROGRAMS)
 	@if test -n "$?" ; then \
